src/main/java/org/apache/flink/benchmark/FlinkEnvironmentContext.java

```diff
@@ -107,6 +110,7 @@ protected Configuration createConfiguration() {
         // It doesn't make sense to wait for the final checkpoint in benchmarks since it only prolongs
         // the test but doesn't give any advantages.
         configuration.set(ExecutionCheckpointingOptions.ENABLE_CHECKPOINTS_AFTER_TASKS_FINISH, false);
+        configuration.set(REQUIREMENTS_CHECK_DELAY, Duration.ZERO);
```

Comment:
nit: after FLINK-28243, we will likely remove this line. So I'd comment here in the code or mention it in FLINK-28243, or both.